diff --git a/node_modules/react-native-gesture-handler/react-native-gesture-handler.d.ts b/node_modules/react-native-gesture-handler/react-native-gesture-handler.d.ts
index 7a1332b..6328045 100644
--- a/node_modules/react-native-gesture-handler/react-native-gesture-handler.d.ts
+++ b/node_modules/react-native-gesture-handler/react-native-gesture-handler.d.ts
@@ -7,11 +7,9 @@ declare module 'react-native-gesture-handler' {
     Animated,
     FlatListProperties,
     ScrollViewProperties,
-    SliderProperties,
     SwitchProperties,
     TextInputProperties,
     ToolbarAndroidProperties,
-    ViewPagerAndroidProperties,
     DrawerLayoutAndroidProperties,
     TouchableHighlightProperties,
     TouchableOpacityProperties,
@@ -409,10 +407,6 @@ declare module 'react-native-gesture-handler' {
     NativeViewGestureHandlerProperties & ScrollViewProperties
   > {}
 
-  export class Slider extends React.Component<
-    NativeViewGestureHandlerProperties & SliderProperties
-  > {}
-
   export class Switch extends React.Component<
     NativeViewGestureHandlerProperties & SwitchProperties
   > {}
@@ -425,10 +419,6 @@ declare module 'react-native-gesture-handler' {
     NativeViewGestureHandlerProperties & ToolbarAndroidProperties
   > {}
 
-  export class ViewPagerAndroid extends React.Component<
-    NativeViewGestureHandlerProperties & ViewPagerAndroidProperties
-  > {}
-
   export class DrawerLayoutAndroid extends React.Component<
     NativeViewGestureHandlerProperties & DrawerLayoutAndroidProperties
   > {}
@@ -443,6 +433,11 @@ declare module 'react-native-gesture-handler' {
     Component: React.ComponentType<any>,
     containerStyles?: StyleProp<ViewStyle>
   ): React.ComponentType<any>;
+
+  export function createNativeWrapper(
+    Component: React.ComponentType<any>,
+    config: NativeViewGestureHandlerProperties
+  ): React.ComponentType<any>;
 }
 
 declare module 'react-native-gesture-handler/Swipeable' {
@@ -482,7 +477,7 @@ declare module 'react-native-gesture-handler/Swipeable' {
 }
 
 declare module 'react-native-gesture-handler/DrawerLayout' {
-  import { Animated, StatusBarAnimation } from 'react-native';
+  import { Animated, StatusBarAnimation, StyleProp, ViewStyle } from 'react-native';
 
   interface DrawerLayoutProperties {
     renderNavigationView: (
